frontend/src/components/common/confirm-modal/ConfirmModal.tsx

```diff
@@ -29,15 +33,34 @@ const ConfirmModal = ({
   onConfirm: () => Promise<any> | any
   onCancel?: () => Promise<any> | any
   disableImage?: boolean
+  needFeedback?: boolean
+  feedbackUrl?: string
 }) => {
   const modalContext = useContext(ModalContext)
+  const { email } = useContext(AuthContext)
   const [errorMessage, setErrorMessage] = useState('')
 
+  // feedback modal is built directly into confirm modal as confirm modal is the component that is commonly used across the campaigns
+  const openFeedbackModal = () => {
+    if (feedbackUrl !== null && needFeedback) {
```

Comment:
Can we use a single arg instead? Cos if `feedbackUrl` is present, it also means we want feedback right

Reply:
good point, have changed to use only 1 arg!